apps/site/src/utils/homepage-demo-src.ts

```
@@ -0,0 +1,20 @@
+import {
+    homepageDemoScenarios,
+    type HomepageDemoLocale,
+    type HomepageDemoScenarioId,
+} from '../data/homepage-demo-scenarios';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)